README.md

```diff
@@ -1,40 +1,42 @@
-# dex-ui
+# DEX UI
 
-## Feedback
+This repo contains an interface to the Superhero decentralized exchange (DEX).
 
-If you simply want to provide feedback, please use the following form: [https://form.jotform.com/221174404956355](https://form.jotform.com/221174404956355)
+The Superhero DEX consists of multiple parts:
+- [UI (this repo)](https://github.com/aeternity/dex-ui)
+- [Backend](https://github.com/aeternity/dex-backend)
+- [Contracts](https://github.com/aeternity/dex-contracts-v2)
 
-For bug reports or feature requests, please use the issues: [https://github.com/aeternity/dex-ui/issues/new/choose](https://github.com/aeternity/dex-ui/issues/new/choose)
+## Access
 
-## Project setup
-```
-npm install
-```
+Find a hosted version of the interface over at [aepp.dex.superhero.com](aepp.dex.superhero.com) or
```

Comment:
that links is broken and shouldn't it be https://dex.superhero.com?

Reply:
ok got it - we have the landingpage and this is the page for the app.

but anyway -> clicking on the link is broken ;-)